runtimes/php/varnish.md

```diff
@@ -2,3 +2,107 @@
 title: Use Varnish as HTTP Cache
 shortdesc: This article shows you how enable Varnish on your application.
 ---
+
+# Varnish
+
+Varnish is a HTTP cache who works as a reverse proxy between the application server and the client, on the server-side.
+Following rules defined by the user, Varnish will caches the data of an applications to reduce the load.
+
+## Enable Varnish for you appliaiton
+
+Varnish is provided by default in PHP instances. To enable it, you just have to create a `varnish.vcl` file
+in the `/clevercloud` folder.
+
+This file describe how Varnish caches your applications and how it decides to return a cached resource or not.
+
+***Warning***: you should not include the `backend` section in your `varnish.vcl` file. We already manage it for you.
```

Comment:
the backend section of the `varnish.vcl` configuration file is not necessary as it is already handled by Clever Cloud